tests/test_views.py

```diff
@@ -88,6 +88,39 @@ def test_search_pagination(self, mock_search):
             index="book",
         )
 
+    @patch("elasticsearch.client.Elasticsearch.search")
+    def test_search_pagination_total_as_dict(self, mock_search):
+        mock_search.return_value = {
+            "_shards": {"failed": 0, "successful": 5, "total": 5},
+            "hits": {
+                "hits": [],
+                "max_score": 0,
+                "total": {"value": 1, "relation": "eq"},
```

Comment:
Hm, what does the "relation" field mean? Could it be we'll get an approximate count?

Reply:
ping?

Reply:
sorry, looks like yes, it can be 
`"relation": "gte"`
and if use request option `track_total_hits` response will be accurate

https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#hits-total-now-object-search-response
> You can force the count to always be accurate by setting track_total_hits to true explicitly in the search request.

Reply:
Hm then it sounds as if we need to set `track_total_hits` to `true` to preserve the existing behaviour, otherwise the totals will be wrong. Could you check whether requests against ES 6 still work correctly when setting this option (i.e. it ignores the parameters)?

Reply:
checked it ignore it
`>>> es.search(body={'track_total_hits': True, 'query': {'bool': {'filter': [{'terms': {'id': [12345]}}]}}})
ok response 

>>> es.info()
{'name': 'iR-5TXu', 'cluster_name': 'es-dev-6', 'cluster_uuid': 'XXX', 'version': {'number': '6.3.1', '